src/reducers/breakpoints.js

```diff
@@ -13,7 +13,7 @@ const { updateObj } = require("../utils/utils");
 const I = require("immutable");
 const makeRecord = require("../utils/makeRecord");
 
-import type { Breakpoint, Location } from "devtools-client-adapters";
+import type { Breakpoint, Location } from "devtools-client-adapters/src/types";
```

Comment:
Maybe we should export all the types from the `devtools-client-adapters/index.js` so you don't have to specify `src/types` here.